packages/crayons-styles/design-tokens/crayons/base.json

```diff
@@ -0,0 +1,237 @@
+        },
+        "100": {
+          "value": "#cfd7df",
+          "type": "color",
+          "var": "--fw-color-secondary-100"
+        },
+        "50": {
+          "value": "#ebeff3",
+          "type": "color",
+          "var": "--fw-color-secondary-50"
+        },
+        "25": {
+          "value": "#f5f7f9",
+          "type": "color",
+          "var": "--fw-color-secondary-25"
+        }
+      },
+      "success": {
+        "800": {
+          "value": "#00795b",
+          "type": "color",
+          "var": "--fw-color-success-800"
+        },
+        "500": {
+          "value": "#00a886",
+          "type": "color",
+          "var": "--fw-color-success-500"
+        },
+        "100": {
+          "value": "#b4e5da",
+          "type": "color",
+          "var": "--fw-color-success-100"
+        },
+        "50": {
+          "value": "#e0f5f1",
+          "type": "color",
+          "var": "--fw-color-success-50"
+        }
+      },
+      "warning": {
+        "700": {
+          "value": "#e86f25",
+          "type": "color",
+          "var": "--fw-color-warning-700"
+        },
+        "300": {
+          "value": "#f8ab59",
+          "type": "color",
+          "var": "--fw-color-warning-300"
+        },
+        "100": {
+          "value": "#fedcb3",
+          "type": "color",
+          "var": "--fw-color-warning-100"
+        },
+        "50": {
+          "value": "#fef1e1",
+          "type": "color",
+          "var": "--fw-color-warning-50"
+        }
+      },
+      "danger": {
+        "900": {
+          "value": "#c82124",
+          "type": "color",
+          "var": "--fw-color-danger-900"
+        },
+        "800": {
+          "value": "#d72d30",
+          "type": "color",
+          "var": "--fw-color-danger-800"
+        },
+        "300": {
+          "value": "#f2797b",
+          "type": "color",
+          "var": "--fw-color-danger-300"
+        },
+        "200": {
+          "value": "#f89fa1",
+          "type": "color",
+          "var": "--fw-color-danger-200"
+        },
+        "100": {
+          "value": "#ffd0d6",
+          "type": "color",
+          "var": "--fw-color-danger-100"
+        },
+        "50": {
+          "value": "#ffecf0",
+          "type": "color",
+          "var": "--fw-color-danger-50"
+        }
+      },
+      "info": {
+        "800": {
+          "value": "#2c5cc5",
+          "type": "color",
+          "var": "--fw-color-info-800"
+        },
+        "100": {
+          "value": "#bbdcfe",
+          "type": "color",
+          "var": "--fw-color-info-100"
+        },
+        "50": {
+          "value": "#e5f2fd",
+          "type": "color",
+          "var": "--fw-color-info-50"
+        }
+      },
+      "neutral": {
+        "100": {
+          "value": "#fff",
+          "type": "color",
+          "var": "--fw-color-neutral-100"
+        }
+      }
+    },
+    "font": {
+      "3xl": {
+        "value": " 2.75rem",
```

Comment:
I see some space before the value. Is it required ?

Reply:
Nope, it's a typo. Fixed it!!